assets/scss/6-components/cta-block/cta-block.html

```diff
@@ -0,0 +1,42 @@
+<!-- note: strong tags in h4s only used for demo -->
+
+<!-- Social Follow CTA -->
+<div class="c-cta-block has-padding has-giant-btm-marg">
+  <div class="c-cta-block__msg t-size-b">
```

Comment:
@ashley-hebler Text here feels a bit large. Can you try swapping for t-size-s? (~14px)